### ISSUE TITLE
Invalid Parameter Error when Running Microsoft-Edge Protocol

### ISSUE THREAD

**Issue by RaspberryKitty1** [rcmaehl/MSEdgeRedirect#470]
### Preflight Checklist

- [ ] Running `microsoft-edge:https://google.com` redirects successfully
- [ ] Microsoft Edge is selected in any "How do you want to open this?" box

### Install Type

Manual Update

### Install Mode

Active Mode

### Steps to reproduce

1. Set up MSEdgeRedirect as described in the documentation. On Windows 11 24H2.
2. On Windows 11 Version 24H2 (OS Build 26100.1742), run the following command in a terminal, Run box, or CMD:
`microsoft-edge:https://google.com`

### ✔️ Expected Behavior

The URL (https://google.com) should be redirected and opened as per the configured behavior in MSEdgeRedirect.

### ❌ Actual Behavior

![image](https://github.com/user-attachments/assets/f9ba71b2-9d8f-43b6-843b-f7ba0eb042a5)

### Microsoft Windows version

Windows 11 Pro 24H2 (OS Build 26100.1742)

**Comment by micwoj92** [comment]
Seems like duplicate of https://github.com/rcmaehl/MSEdgeRedirect/issues/403 however the error message is slightly different.

**Comment by rcmaehl** [comment]
@RaspberryKitty1 Is Microsoft Edge still installed?

**Comment by RaspberryKitty1** [comment]
I don't believe Edge is installed, as I recall using the tool mentioned in the installer to remove it. That said, everything was working fine prior to the 24H2 update for Windows 11. Unfortunately, I won't be able to confirm how Edge was removed until I have access to my PC again, which will be late Friday

**Comment by RaspberryKitty1** [comment]
> @RaspberryKitty1 Is Microsoft Edge still installed? If not, how was it removed?

1st I wanna apologize I got the mode it's installed in incorrectly. it's installed in Active mode, not Service Mode.
2nd 
![image](https://github.com/user-attachments/assets/89d8b21a-1daa-4f27-b491-e051cf7f2a2f)

**Comment by RaspberryKitty1** [comment]
I got it to work.

After some trial and error, here's what I did:

Reinstalled Microsoft Edge.

Reran AveYo's Edge Remover (though it doesn’t seem to fully uninstall Edge on Windows 11 24H2).

Reinstalled MSEdgeRedirect.


Everything is working now, but I’m stuck with Edge installed even though I don’t want it.

**Comment by RaspberryKitty1** [comment]
I got it to work.

After some trial and error, here's what I did:

Reinstalled Microsoft Edge.

Reran AveYo's Edge Remover (though it doesn’t seem to fully uninstall Edge on Windows 11 24H2).

Reinstalled MSEdgeRedirect.


Everything is working now, but I’m stuck with Edge installed even though I don’t want it.

**Comment by rcmaehl** [comment]
Yeah, I think there was a bug report on AveYo's github repo about 24H2. I'll look into fixing this so it'll still work with Edge Uninstalled with methods other than his script. The main issue will be figuring out how to get executables to use as I can't legally "redistribute" the microsoft whitelisted executables.
